src/services/groups/index.ts

```diff
@@ -10,6 +10,7 @@ import { Paginated } from "@/models/pagination";
 import { revalidatePath } from "next/cache";
 import { ScimReference, User } from "@/models/scim";
 import { setNotification } from "@/components/toaster";
+import { fetchMe } from "@/services/me";
```

Comment:
This stinks! We are coupling two services that should be independent from each other